test/compilable/test18348.sh

```diff
@@ -0,0 +1,35 @@
+#!/usr/bin/env bash
+
+set -eu pipefail
+
+dir=${RESULTS_DIR}${SEP}compilable
+output_file=${dir}/test18348.sh.out
+
+faketty () { script -qfc "$(printf "%q " "$@")" /dev/null ; }
+export TERM="faketerm"
```

Comment:
Apparently this trick doesn't on some CI. Setting this to blocked as https://github.com/dlang/dmd/pull/7823 is a lot nicer.